builds/misc/images.yaml

```
@@ -284,6 +291,13 @@ jobs:
           imageName: azureiotedge-deployment-tester
           project: DeploymentTester
 
+     # Cloud To Device Message Tester
+      - template: templates/image-linux.yaml
+        parameters:
+          name: Cloud To Device Message Tester
+          imageName: azureiotedge-cloud-to-device-message-tester
```

nit: how about below suggestion to shorten image name?
```suggestion
          imageName: azureiotedge-c2dmessage-tester
```